spec/onchain.tex

```diff
@@ -339,17 +397,21 @@ \subsection{Decrement Transaction}\label{sec:decrement-tx}
 	\[
 		(\stOpen,\cid,\hydraKeys,\cPer,v,\eta) \xrightarrow[\xi, s, m]{\mathsf{decrement}} (\stOpen,\cid,\hydraKeys,\cPer,v',\eta')
 	\]
+	\item Decrement snapshot number $s'$ is higher than the currently stored snapshot number $s$
+	\[
+		s' > s
+	\]
 	\item New version $v'$ is incremented correctly
 	\[
 		v' = v + 1
 	\]
-	\item $\xi$ is a valid multi-signature of the currency id $\cid$, the current state version $v$, and the new state $\eta'$
+	\item $\xi$ is a valid multi-signature of the new snapshot state (currency id $\cid$, the current state version $v$, the new state $\eta'$ \red{, UTxO to deposit $\eta_\alpha$} and UTxO to withdraw $\eta_\omega$)
 	\[
-		\msVfy(\hydraKeys,(\cid || v || s || \eta' || \eta_{\omega}),\xi) = \true
+		\msVfy(\hydraKeys,(\cid || v || s || \eta' || \red{\eta_\alpha} || \eta_\omega),\xi) = \true
 	\]
-	where snapshot number $s$ is provided through the redeemer and $\eta_{\omega}$ must be the digest of all removed UTxO
+    where snapshot number $s$ is provided through the redeemer \red{, $\eta_\alpha$ is a digest of all added UTxO (in case of decrement transaction $\eta_\alpha = \bot$)} and $\eta_\omega$ must be the digest of all removed UTxO.
```

Comment:
Should: Just put $\bot$ in the expression and don't use $\eta_\alpha$ (simplifies definition IMO)